.gometalinter.json

```diff
@@ -0,0 +1,34 @@
+{
+  "Vendor": true,
+  "Deadline": "5m",
+  "Concurrency": 2,
+  "Linters": {
+    "gofmt": {"Command": "gofmt -l -s -w"},
+    "goimports": {"Command": "goimports -l -w"}
```

Comment:
This is just for convenience. It adds the `-w` flags for these two commands so that if you run it locally, and there are  errors, the fixes for the errors will be written.